keras_cv/models/backbones/mix_transformer/mix_transformer_backbone.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""MiT backbone model.
+
+References:
```

Comment:
Maybe this is a TODO?

Let's add the reference and move the docstring to live inside the `MiTBackbone` class

Reply:
Yup, left as a TODO. By inside the `MiTBackbone` class - you mean within the class def? It looks like references are outside of the classes for other backbones?

Reply:
Yeah, for the base `Backbone` models we've been including the docstring in the class directly (e.g. [here](https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/backbones/densenet/densenet_backbone.py#L41))